static/js/tools/scatter/util.ts

```diff
@@ -73,6 +73,41 @@ async function getStatsWithinPlace(
   });
 }
 
+export async function getAllStatsWithinPlace(
```

Comment:
please add documentation, especially for shared functions

Reply:
done